var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -154,16 +154,21 @@ def run_cxx_example_test(self):
 
         test_dir = join_path(self.test_suite.current_test_cache_dir, 'examples', 'apps')
 
-        if not os.path.exists(test_dir):
-            print('Skipping caliper test')
+        if not os.path.isfile(join_path(test_dir, 'cxx-example.cpp')):
+            print('Skipping caliper test: file does not exist')
```

Comment:
I don't know if it matters, but here you can probably use tty.msg so it shows up in spack colors, etc. Or if it's debug, tty.debug.